src/components/analyses/details/Drawer.js

```diff
@@ -108,16 +99,22 @@ function DetailsDrawer(props) {
                 variant: "outlined",
             }}
         >
-            <div className={classes.drawerHeader}>
-                <Typography variant="h6">{analysisName}</Typography>
-            </div>
+            <Typography variant="h6" className={classes.drawerHeader}>
+                {analysisName}
+            </Typography>
```

Comment:
Just for my own edification, did removing the div have any effect on the page display format?

Reply:
Nope, in this case, the wrapping `div` was extraneous, and these `drawerHeader` styles can be applied directly to the `Typography` component (which will render as an `h6` HTML element here).